### PR TITLE
Update eyecare-e-referrals-service.yaml

### PR DESCRIPTION
**Pull request by mick-sch**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [x] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/eyecare-e-referrals-service.yaml [hunk 1]
```diff
@@ -15,36 +15,40 @@ info:
     email: api.management@nhs.net
   description: >-
     ## Overview
-    Use this API to pass details of a new referral from an Optomoetry Patient Management System (Optometry PMS) to an Eyecare e-referral system (EeRS).
-    The goal of the API is to pre-populate clinical, user and patient information from Optometry PMS systems to an EeRS  – saving clinicians time and improving patient safety.    
+    
+    Use this API standard to pass details of a new referral from an Optomoetry Patient Management System (Optometry PMS) to an Eyecare e-referral system (EeRS).
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:19]: OK

File: specification/eyecare-e-referrals-service.yaml [hunk 2]
```diff
@@ -15,36 +15,40 @@ info:
...
+    
+    The goal of the API standard is to pre-populate clinical, user and patient information from an Optometry PMS systems to an EeRS  – saving clinicians time and improving patient safety.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:21]: OK

File: specification/eyecare-e-referrals-service.yaml [hunk 3]
```diff
@@ -15,36 +15,40 @@ info:
...
+    
+    Use this API standard to pass details of a new referral from an Optomoetry Patient Management System (Optometry PMS) to an Eyecare e-referral system (EeRS).
+    
+    The goal of the API standard is to pre-populate clinical, user and patient information from an Optometry PMS systems to an EeRS  – saving clinicians time and improving patient safety.    
+    
     The API standard supports auto-population of GOS18 and associated referral data from Optical IT systems. This is important to engaging Optometrists in digital referrals into the NHS, and eliminate double-keying of information documented as part of a sight test or optical consultation.
     
-    This API is a published specification only.  The API is implemented and exposed by each EeRS system.  Therefore implementing the API and integrating is done directly with each of the EeRS systems, and not via NHS Digital services.
+    This API standard is a published specification only. You implement and expose it with each EeRS system.  Therefore you implement and integrate it directly with each of the EeRS systems, and not via NHS Digital services.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:25]: OK

File: specification/eyecare-e-referrals-service.yaml [hunk 4]
```diff
@@ -15,36 +15,40 @@ info:
...
     
     You can:
-    Transfer key clinical, patient and practitioner information associated with making a referral to NHS health services via an EeRS. 
-    The API standard supports auto-population of GOS18 and associated referral data from the Optometry PMS. 
+    - transfer key clinical, patient and practitioner information associated with making a referral to NHS health services via an EeRS
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:28]: OK

File: specification/eyecare-e-referrals-service.yaml [hunk 5]
```diff
@@ -15,36 +15,40 @@ info:
...
+    - auto-populate GOS18 and associated referral data from the Optometry PMS
     
     ## Who can use this API
-    This API can be used by any Optometry PMS to allow an Optometrist to send a referral into a specific EeRS system. Integrated Care Systems will request that each Optometry system in use in their area links to their specific EeRS system.
+    
+    This API standard can be used by any Optometry PMS to allow an Optometrist to send a referral into a specific EeRS system. Integrated Care Systems will request that each Optometry system in use in their area links to their specific EeRS system.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:33]: OK

File: specification/eyecare-e-referrals-service.yaml [hunk 6]
```diff
@@ -15,36 +15,40 @@ info:
...
-    3). Ask the EeRS supplier you are integrating with for a system to system API key 
-    4). Ask the EeRS supplier for an API endpoint
-    5). Develop your system to connect to the API and send the API key with the data payload to that EeRS endpoint 
-    6). Test that the script and launch process are working with the EeRS supplier, and check the right data fields are coming across with a clinician 
-    7). Go live with the API for the practices that are licensed for use of the EeRS supplier in question
-    8). Maintain a log of which sites are using which EeRS systems. EeRS suppliers should be able to confirm this at the practice level for you
+    1. Identify the data fields in your system that correspond to the data fields in the specification.
+    2. Create a way of initiating the API – ideally using a “refer via EeRS” button in the right place in your PMS system for the stores that are associated with that EeRS referral platform.
+    3. Ask the EeRS supplier you are integrating with for a system to system API key.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:46]: OK

File: specification/eyecare-e-referrals-service.yaml [hunk 7]
```diff
@@ -15,36 +15,40 @@ info:
...
-    3). Ask the EeRS supplier you are integrating with for a system to system API key 
-    4). Ask the EeRS supplier for an API endpoint
-    5). Develop your system to connect to the API and send the API key with the data payload to that EeRS endpoint 
-    6). Test that the script and launch process are working with the EeRS supplier, and check the right data fields are coming across with a clinician 
-    7). Go live with the API for the practices that are licensed for use of the EeRS supplier in question
-    8). Maintain a log of which sites are using which EeRS systems. EeRS suppliers should be able to confirm this at the practice level for you
+    1. Identify the data fields in your system that correspond to the data fields in the specification.
+    2. Create a way of initiating the API – ideally using a “refer via EeRS” button in the right place in your PMS system for the stores that are associated with that EeRS referral platform.
+    3. Ask the EeRS supplier you are integrating with for a system to system API key. 
+    4. Ask the EeRS supplier for an API endpoint.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:47]: OK

File: specification/eyecare-e-referrals-service.yaml [hunk 8]
```diff
@@ -15,36 +15,40 @@ info:
...
+    5. Develop your system to connect to the API and send the API key with the data payload to that EeRS endpoint. 
+    6. Test that the script and launch process are working with the EeRS supplier, and check the right data fields are coming across with a clinician.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:49]: Good question, don't know.

File: specification/eyecare-e-referrals-service.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:28]: To investigate later.

File: specification/eyecare-e-referrals-service.yaml [hunk 9]
```diff
@@ -15,36 +15,40 @@ info:
...
+    - auto-populate GOS18 and associated referral data from the Optometry PMS
     
     ## Who can use this API
-    This API can be used by any Optometry PMS to allow an Optometrist to send a referral into a specific EeRS system. Integrated Care Systems will request that each Optometry system in use in their area links to their specific EeRS system.
+    
+    This API standard can be used by any Optometry PMS to allow an Optometrist to send a referral into a specific EeRS system. Integrated Care Systems will request that each Optometry system in use in their area links to their specific EeRS system.
     
     The EeRS systems are procured by each NHS region from the [Dynamic Purchasing System (DPS) for Electronic Eyecare Referral Systems (EeRS)](https://www.nhsx.nhs.uk/key-tools-and-info/procurement-frameworks/dynamic-purchasing-system-electronic-eyecare-referral-systems/). 
-    Each if these systems is a receiver of referrals via this API and has their own endpoint for the service.
+    Each if these systems receives referrals via this API standard and has their own endpoint for the service.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:36]: Good spot.

File: specification/eyecare-e-referrals-service.yaml [hunk 10]
```diff
@@ -15,36 +15,40 @@ info:
...
     
-    Therefore, sending Optometry PMS systems will need to link to each EeRS eyecare e-referral supplier's implementation of the API.  Also each endpoint will require to be secured via an APIKEY provided by that supplier.
+    Therefore, the sending Optometry PMS system needs to link to each EeRS eyecare e-referral developer's implementation of the API standard.  Also each endpoint must be secured via an APIKEY provided by that developer.
```
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:38]: Agree needs clarification by owner..

File: specification/eyecare-e-referrals-service.yaml (hunk 8 above)
[the remark this answers is not in the document]
Comment (mick-sch) [reply specification/eyecare-e-referrals-service.yaml:49]: To be resolved Clinician versus Practitioner
To be resolved supplier versus developer (developer assumed)

**Review by mick-sch — COMMENTED** [review]
Ready for 2nd PR review.

**Review by SachinPath — APPROVED** [review]

**Pull request merged**